src/specs/modules/NativeCustomerIO.ts

```diff
@@ -25,7 +27,7 @@ import { TurboModuleRegistry, type TurboModule } from 'react-native';
  *
  * @internal - Not exported to avoid conflicts with public API types
  */
-type NativeBridgeObject = Object;
+type NativeBridgeObject = UnsafeObject;
```

Comment:
Not much different from `Object` as `UnsafeObject=object`, just used type from codegen, generated code stays the same